core/field_checkbox.js

```diff
@@ -42,15 +42,33 @@ goog.require('Blockly.utils.Size');
  *    changes to the field's value. Takes in a value ('TRUE' or 'FALSE') &
  *    returns a validated value ('TRUE' or 'FALSE'), or null to abort the
  *    change.
+ * @param {Object=} opt_config A map of options used to configure the field.
+ *    See the documentation for a list of properties this parameter supports.
+ *    https://developers.google.com/blockly/guides/create-custom-blocks/fields/built-in-fields/checkbox#creation
  * @extends {Blockly.Field}
  * @constructor
  */
-Blockly.FieldCheckbox = function(opt_value, opt_validator) {
+Blockly.FieldCheckbox = function(opt_value, opt_validator, opt_config) {
+  /**
+   * Character for the check mark. Used to apply a different check mark
+   * character to individual fields.
+   * @type {?string}
+   * @private
+   */
+  this.checkChar_ = null;
+
+  if (opt_config) {
+    if (opt_config['checkCharacter']) {
```

Comment:
Just so we're consistent, lets move this to a separate configure method like the other PR, and I think this is good to go. We can fix the ordering of the constructor in a separate PR.